multiapps-controller-client/src/main/java/org/cloudfoundry/multiapps/controller/client/facade/rest/CloudControllerRestClientImpl.java

```diff
@@ -393,11 +395,29 @@ private void updateApplicationProcess(UUID applicationGuid, Staging staging) {
         if (staging.getHealthCheckType() != null) {
             updateProcessRequestBuilder.healthCheck(buildHealthCheck(staging));
         }
+        if (staging.getReadinessHealthCheckType() != null) {
+            updateProcessRequestBuilder.readinessHealthCheck(buildReadinessHealthCheck(staging));
+        }
         delegate.processes()
                 .update(updateProcessRequestBuilder.build())
                 .block();
     }
 
+    private ReadinessHealthCheck buildReadinessHealthCheck(Staging staging) {
+        return ReadinessHealthCheck.builder()
+                                   .type(ReadinessHealthCheckType.from(staging.getReadinessHealthCheckType()))
+                                   .data(buildReadinessHealthCheckData(staging))
+                                   .build();
+    }
+
+    private Data buildReadinessHealthCheckData(Staging staging) {
+        return Data.builder()
+                   .invocationTimeout(staging.getReadinessHealthCheckTimeout())
```

Comment:
As parameter name in mta descriptor, is it used same name as in the API - invocation timeout?